0000-interledger-eli5/0000-interledger-eli5.md

```diff
@@ -0,0 +1,124 @@
+
+Setup goes like this:
+
+1) A sender asks a connector: "Do you know this receiver?"
+
+2) If the connector knows the receiver, great.  If not, the connector talks to its "peers" (its "connector friends") to see if any know the receiver, or know another connector that might know the receiver... and so on.  Through this process, a chain of connectors linking the sender to the receiver is discovered.  This chain may involve just one connector, or it may involve many.
+
+3) Connectors can each charge a little bit for their part in handling the payment.  After the chain is built, the connector closest the sender explains how much the receiver will get if the payment goes down that chain.  If it's much less than what the sender wants to send, the sender will probably choose a different connector.  So connectors try make the cost to the sender as little as possible, while still earning something.
+
+4) The sender decides whether or not to use that connector, based on how much it will cost.  If it's a "no", the sender can start over asking a different connector, or asking the same one again (who may figure out a better way this time).  When the sender is happy with the cost, the payment can go ahead.
+
+For each payment, a chain with links of trust is formed between the sender, connector - or connectors - and the receiver.  Each link in the chain trusts the one before it not to waste its time or run away with the assets, but no link needs to trust any other links in the chain or even know about them.  What's great about this, is it means the sender and the receiver don't need to trust each other, or even know who each other is.  
+
+_Key point: If a sender makes a payment with ILP, it is assured that either the receiver will receive the amount the sender agreed to, or the sender will get their original amount returned.  The sender doesn't need to trust anyone in-between, and the receiver can be sure the sender agreed to the transfer (and so won't later try get the asset back).  This is what makes ILP payments "trustless"._
+
+**Payment Security**
+
+Even with all the preparation, there are still a couple of things that could go wrong.  For example, what happens if a connector lies? There are a couple of ways connectors could choose to either be annoying and cause delays, or even try and steal assets.  It's unlikely, but if it did happen, ILP guarantees the sender will get their original asset back in their account.  Then of course, the sender could try sending again a different way.  In a small number of cases, this situation may result in a connector next to the bad one losing money.  That's why connectors charge a little bit for making payments, and it's also why they want to choose carefully to link up with only good connectors.  Also, if a connector did lie, all connectors will know about it and "unfriend" them to make it difficult for that connector to be involved in an ILP payment again.
```

Comment:
In the case where you have proper ledgers in between each pair of connectors, connectors can only get screwed by the ledger, not by other connectors